exporter/api/query_handlers.go

```diff
@@ -66,31 +65,30 @@ func HandleParticipantsQuery(logger *zap.Logger, qbftStorage *storage.QBFTStores
 		return
 	}
 
-	beaconRole, err := message.BeaconRoleFromString(nm.Msg.Filter.Role)
+	role, err := message.BeaconRoleFromString(nm.Msg.Filter.Role)
 	if err != nil {
 		logger.Warn("failed to parse role", zap.Error(err))
 		res.Data = []string{"role doesn't exist"}
 		nm.Msg = res
 		return
 	}
-	runnerRole := casts.BeaconRoleToConvertRole(beaconRole)
-	roleStorage := qbftStorage.Get(runnerRole)
+	roleStorage := store.Get(role)
 	if roleStorage == nil {
-		logger.Warn("role storage doesn't exist", fields.ExporterRole(runnerRole))
-		res.Data = []string{"internal error - role storage doesn't exist", beaconRole.String()}
+		logger.Warn("role storage doesn't exist", fields.ExporterRole(role))
+		res.Data = []string{"internal error - role storage doesn't exist", role.String()}
 		nm.Msg = res
 		return
 	}
 
-	msgID := convert.NewMsgID(domain, pkRaw, runnerRole)
 	from := phase0.Slot(nm.Msg.Filter.From)
 	to := phase0.Slot(nm.Msg.Filter.To)
-	participantsList, err := roleStorage.GetParticipantsInRange(msgID, from, to)
+	participantsList, err := roleStorage.GetParticipantsInRange(role, spectypes.ValidatorPK(pkRaw), from, to)
```

Comment:
need length validation above in pkRaw otherwise this conversion may cause issues

Reply:
added the check